docs/source/specs/openapi.json

```diff
@@ -3326,6 +3335,7 @@
       "CrossAccountRequestIn": {
         "required": [
           "target_account",
+          "target_org",
```

Comment:
We're not making this a required field to start with, right? It's optional, so we should leave it out of the `required` fields (for now) otherwise it will force clients to send it.